### PR TITLE
[Fix] attackiq-get-test-results missing output paths

### PR DESCRIPTION
**Pull request by DeanArbel**
## Status
- [x] Ready

## Related Issues:
fixes: https://github.com/demisto/etc/issues/24281

## Description
Fixed `Job State` and `Assessment` outputs in the command `attackiq-get-test-results`.

## Does it break backward compatibility?
   - [x] No

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

File: Packs/AttackIQFireDrill/Integrations/AttackIQFireDrill/AttackIQFireDrill.py [hunk 1]
```diff
@@ -263,7 +263,7 @@ def get_page_number_and_page_size(args):
 ''' COMMANDS + REQUESTS FUNCTIONS '''
 
 
-def test_module():
+def module_test():
```
Comment (yuvalbenshalom) [inline Packs/AttackIQFireDrill/Integrations/AttackIQFireDrill/AttackIQFireDrill.py:266]: why did you change the name?

Comment (DeanArbel) [reply Packs/AttackIQFireDrill/Integrations/AttackIQFireDrill/AttackIQFireDrill.py]: The ide sometimes recognizes functions that start with `test_` as test functions that should be ran with pytest, this caused me some odd behaviors while debugging. Either way, I reverted this, as it's not required.
